extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -398,17 +484,14 @@ protected function getImage( $wikiId, $articleId ) {
 			if ( $row ) {
 				$dbName = $row->city_dbname;
 				if ( !empty( $dbName ) ) {
-					$db = wfGetDB( DB_SLAVE, [ ], $dbName ); // throws if database does not exits.
+					$db = wfGetDB( DB_SLAVE, [], $dbName ); // throws if database does not exits.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)